src/d3a/d3a_core/sim_results/endpoint_buffer.py

```diff
@@ -84,7 +84,7 @@ def generate_json_report(self):
             "tree_summary": self.tree_summary,
             "status": self.status,
             "device_statistics": self.device_statistics_time_str_dict,
-            "energy_trade_profile": self.energy_trade_profile
+            "energy_trade_profile": self.energy_trade_profile_redis
```

Comment:
Do we really want to expose the redis endpoint for the energy_trade_profile ?

Reply:
No, this is a test remnant. Will revert it. Thanks!